Sources/OpenTelemetrySdk/Metrics/Export/MetricExporter.swift

```diff
@@ -19,12 +19,48 @@ public enum MetricExporterResultCode {
     case success
     case failureNotRetryable
     case failureRetryable
+
+    /// Merges the current result code with other result code
+    /// - Parameter newResultCode: the result code to merge with
+    public mutating func mergeResultCode(newResultCode: MetricExporterResultCode) {
```

Comment:
does it have to be public?

Reply:
No, it does not. Changed this and also the one in SpanExporterResultCode.  Thanks a lot.